apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -1,4175 +1,798 @@
-element
-elements
 eleven
-eliminate
-eliminated
 else
 elsewhere
-em
-email
-emerged
-emergency
-emission
-emphasis
-emphasize
-empirical
-employed
-employee
-employees
-employment
-empty
-enable
-encounter
-encountered
-encourage
-encouraged
-encouraging
-end
-ended
-ending
-endless
-ends
-enemies
-enemy
-energy
-enforced
-enforcement
-engaged
-engagement
-engine
-engineer
-engineering
-engineers
-enjoy
-enjoyed
-enjoyment
-enormous
 enough
-ens
-enter
-entered
-entering
-enterprise
-entertainment
-enthusiasm
-enthusiastic
-entire
 entirely
-entitled
-entrance
-entries
-entry
-envelope
-environment
-equal
-equally
-equate
-equation
-equipment
-equipped
-equivalent
-era
-error
-errors
-escape
-esp
 especially
-essential
-essentially
-est
-establish
-established
-establishing
-establishment
-estate
-estimate
-estimated
-estimates
 et
+et-al
 etc
-etc.
-eternal
-ethical
-ethics
-evaluation
-eve
 even
-evening
 evenly
-event
-events
-eventually
 ever
 evermore
 every
 everybody
 everyone
 everything
 everywhere
-evidence
-evident
-evidently
-evil
-ex
-exact
 exactly
-examination
-examine
-examined
 example
-examples
-excellent
 except
-exception
-exceptions
-excess
-excessive
-exchange
-excite
-excited
-excitement
-exciting
-exclusive
-exclusively
-excuse
-executive
-exercise
-exercises
-exhibit
-exhibition
-exist
-existed
-existence
-existing
-exists
-expanded
-expanding
-expansion
-expect
-expectations
-expected
-expects
-expenditures
-expense
-expenses
-expensive
-experience
-experienced
-experiences
-experiment
-experimental
-experiments
-expert
-experts
-explain
-explained
-explains
-explanation
-explicit
-exploration
-exposed
-exposure
-express
-expressed
-expressing
-expression
-extend
-extended
-extending
-extension
-extensive
-extent
-extra
-extraordinary
-extreme
-extremely
-eye
-eyes
 f
-fabrics
-face
-faced
-faces
-facilities
-facing
-fact
-factor
-factories
-factors
-factory
-facts
-faculty
-fail
-failed
-failure
-faint
-fair
-fairly
-faith
-fall
-fallen
-falling
-fallout
-familiar
-family
-famous
-fans
-fantastic
-far
-farm
-farmer
-farmers
-farther
-fascinating
-fashion
-fast
-fat
-fate
-father
-fathers
-fault
-favor
-favorable
-favorite
-fear
-fears
-feature
-features
-feb
-february
-fed
-federal
-feed
-feel
-feeling
-feelings
-feels
-fees
-feet
-fell
-fellow
-felt
-female
-fence
-festival
-few
-fewer
-fiber
-fibers
-fiction
-field
-fields
+fi
 fifteen
 fifth
 fifty
-fig
-fight
-fighting
-figure
-figured
-figures
-file
-filed
-filing
-fill
-filled
-filling
-film
-films
-final
-finally
-finance
-financial
-financing
+fify
 find
-finding
-findings
-finds
-fine
-finger
-fingers
-finish
-finished
-fire
-fired
-firing
-firm
-firmly
-firms
 first
-fiscal
-fish
-fishing
-fist
-fit
-fitted
 five
-fixed
-flagicon
-flash
-flat
-fled
-flesh
-flew
-flexible
-flight
-floor
-flow
-flower
-flowers
-fluid
-flux
-fly
-flying
-foam
-foams
-focus
-fog
-folk
-folks
-follow
-followed
-following
-follows
-font
-food
-foods
-fool
-foot
-football
+fix
```

Comment:
I'd skip `fix` as well, at least for the initial pass of stopwords. Someone might be researching into someone's attempts to "fix" the economy for example. If "fix" is a stopword then a lot of other words from the removed list should belong in the list of stopwords as well.